node/src/BaseClient.ts

```diff
@@ -1948,6 +1950,124 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Computes the intersection of sorted sets given by the specified `keys` and returns a list of intersecting elements.
+     * To get the scores as well, see `zinterWithScores`.
+     * To store the result in a key as a sorted set, see `zinterStore`.
+     *
+     * When in cluster mode, all keys in `keys` must map to the same hash slot.
+     * 
```

Comment:
Please add `since` section to all new commands

Reply:
Is it necessary? We refer to the web page for more information about it, as it appears in the valkey document.

Reply:
We still officially support redis 6+, but this command was added in 6.2.
We need to inform user about that and ideally check redis version in tests, because they could be (and should be) executed on redis 6.0 too.